app/views/pages/home.html.erb

```diff
@@ -1,71 +1,134 @@
+  </div>
+  <% if @overview.organization %>
+  <!-- organization section -->
+  <div class="row p-3 p-md-5">
+    <div class="row text-center">
+      <h2 class="d-flex justify-content-center align-items-center">
+        <% if @overview.organization.icon.attached? %>
+          <%= image_tag(@overview.organization.icon, height: 32, class:'me-2') %>
+        <% end %>
+        <%= @overview.organization.name %>
+      </h2>
+      <% if @overview.last_org_files.present? %>
+      <p>
+        <%= t('.organization.last_upload.one', upload_time: time_ago_in_words(@overview.last_org_files.last.created_at.to_time)) %>
+      </p>
+      <% elsif @overview.organization.provider? %>
+      <p>
+        <%= t('.organization.last_upload.zero') %>
+      </p>
+      <% end %>
     </div>
-  </div> -->
-
-
-  <div class="row rows-cols-2 w-100 my-md-3 pl-md-3 border-top">
-    <div class="col pt-3 px-3 pt-md-5 px-md-5 text-center overflow-hidden">
-      <div class="my-3 py-3">
-        <h2 class="display-5">Read the documentation</h2>
-        <ul class="list-group lead align-left">
-          <li class="list-group-item border-0">
-            <%= link_to 'Creating and managing API tokens', 'https://github.com/pod4lib/aggregator/wiki/Creating-and-managing-API-tokens', class: 'text-muted' %>
-          </li>
-          <li class="list-group-item border-0">
-            <%= link_to 'Data requirements', 'https://github.com/pod4lib/aggregator/wiki/Data-requirements', class: 'text-muted' %>
+    <div class="mt-3 mt-md-5 d-md-flex justify-content-md-around">
+      <% unless @overview.last_org_files.empty? %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.uploads.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% @overview.last_org_files.each do |file| %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.metadata_status[file.pod_metadata_status].icon_class, class: "pod-metadata-status #{file.pod_metadata_status} me-2") %>
+            <%= file.filename %>
           </li>
-          <li class="list-group-item border-0">
-            <%= link_to 'Uploading data using the API', 'https://github.com/pod4lib/aggregator/wiki/Uploading-data-using-the-API', class: 'text-muted' %>
+          <% end %>
+          <li class="my-3"><%= link_to t('.organization.uploads.all'), organization_stream_path(@overview.organization, @overview.organization.default_stream) %></li>
+        </ul>
+      </div>
+      <% end %>
+      <% if @overview.organization.provider? %>
+      <div class="my-2 my-md-0">
+        <h3>
+          <%= t('.organization.jobs.heading') %>
+        </h3>
+        <ul class="list-unstyled">
+          <% if @overview.active_org_jobs.present? %>
+          <% @overview.active_org_jobs.each do |status, jobs| %>
+          <li class="my-3">
+            <%= bootstrap_icon(Settings.job_status_group[status].icon_class, class: "pod-icon pod-job-tracker-status #{status} me-2")%>
```

Comment:
Might be nice if the needs attention icon displayed in yellow like it does in other contexts.

Reply:
Good catch; I think the easiest way to fix that was just adding another option to the CSS. Should be good now